renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -435,12 +435,7 @@ __attribute__((visibility("default"))) EGLContext eglCreateContext(EGLDisplay di
   init.depthBits = value;
   eglhooks.eglGetConfigAttrib_real(display, config, EGL_STENCIL_SIZE, &value);
   init.stencilBits = value;
-// TODO: how to detect this?
-#if ENABLED(RDOC_ANDROID)
   init.isSRGB = 0;
```

Comment:
`EGL_KHR_gl_colorspace` sounds like it's a match for what this code is intended for - for sRGB support for GL-created textures we can capture that normally with the format, this `isSRGB` flag is to mark if the default framebuffer is sRGB. That way when we create a fake stand-in texture we know the format to create.

I'm fine with this as a default but maybe a comment should be left marking a future improvement to detect use of `EGL_KHR_gl_colorspace` and set this bool?